learning/tour-of-beam/frontend/lib/pages/welcome/screen.dart

```diff
@@ -226,8 +230,9 @@ class _IntroText extends StatelessWidget {
                     .bodyLarge!
                     .copyWith(color: Theme.of(context).primaryColor),
                 recognizer: TapGestureRecognizer()
-                  ..onTap = () {
-                    // TODO(nausharipov): sign in
+                  ..onTap = () async {
+                    // TODO(nausharipov): popup with auth methods?
```

Comment:
Yes.

Reply:
@alexeyinkin can it be added to gitignore?